app/src/app/classes/stix/technique.ts

```diff
@@ -116,8 +117,9 @@ export class Technique extends StixObject {
                 rep.stix.x_mitre_effective_permissions = this.effective_permissions;
             }
             if (this.tactics.includes('defense-evasion')) rep.stix.x_mitre_defense_bypassed = this.defense_bypassed;
-            if (this.tactics.includes('execution')) rep.stix.x_mitre_remote_support = this.remote_support;
+            if (this.tactics.includes('exfiltration')) rep.stix.x_mitre_remote_support = this.supports_remote;
             if (this.tactics.includes('impact')) rep.stix.x_mitre_impact_type = this.impact_type;
+            if (this.tactics.includes('privilege-escalation')) rep.stix.x_mitre_network_requirements = this.requires_network;
```

Comment:
I'm a bit confused on these changes. I thought the mapping of `execution` to `supports_remote` was correct, just needed a rename. Should it be `exfiltration -> requires_network` and `execution -> supports_remote`?